omod/src/main/java/org/openmrs/module/adminui/page/controller/systemadmin/accounts/AccountPageController.java

```diff
@@ -203,7 +254,21 @@ public void setModelAttributes(PageModel model, Account account, OtherAccountDat
 		model.addAttribute("allowedLocales", administrationService.getAllowedLocales());
 		List<ProviderRole> providerRoles = providerManagementService.getAllProviderRoles(false);
 		model.addAttribute("providerRoles", providerRoles);
-		
+
+		List<Extension> customPersonFragments = appFrameworkService.getExtensionsForCurrentUser("createUser.manageAccountPersonFragment");
+		Collections.sort(customPersonFragments);
```

Comment:
Since the other one one has "View", shouldn't this have "Edit"?

Reply:
replaced with "Edit"